test-engine/src/main/java/gov/nist/toolkit/testengine/transactions/BasicTransaction.java

```diff
@@ -773,6 +787,8 @@ protected void parseGatewayEndpoint(String home, boolean isSecure) throws Except
 				}
 		}
 		testLog.add_name_value(instruction_output, "Endpoint", endpoint);
+		System.out.println("\n\n Home " + home + "\n\n");
```

Comment:
The System.out.println should be changed to a logger. info/fine/finer method call.

Reply:
I committed a new version of the file that removes that line. The log is not needed.